onadata/apps/logger/migrations/0060_auto_20200305_0357.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.9 on 2020-03-05 08:57
+
+from django.db import migrations, models
```

Comment:
I'm a bit conflicted on whether this should be in this PR. Feels like this index will be helpful in the future (FLOIP endpoint uses the `uuid` field quite a bit)